<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyChat</class>
 <widget class="QMainWindow" name="PyChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyChat</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="setting_FRM">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>600</y>
      <width>381</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bk_sett_LBL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>1300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_bk_BTN">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">pushButton&gt;setStyleSheet(&quot;QPushButton{background:transparent;}&quot;);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/menu (copy).png</normaloff>../../icons/menu (copy).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="profile_LBL">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>140</y>
       <width>180</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>382</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 193, 165);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>63</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#eeeeec;&quot;&gt;Setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>bk_sett_LBL</zorder>
    <zorder>profile_LBL</zorder>
    <zorder>label_4</zorder>
    <zorder>menu_bk_BTN</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QFrame" name="mother_FRM">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="record_b">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>510</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/radio.png</normaloff>../../icons/radio.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="background_l">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>691</width>
       <height>451</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="attach_b_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="call_b">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="attach_b">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>321</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>661</width>
       <height>431</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>657</width>
        <height>427</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>      What's you are looking for ?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_b">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>add user</string>
     </property>
    </widget>
    <widget class="QPushButton" name="emoji_BTN">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>513</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/radio.png</normaloff>../../icons/radio.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="user_search_b">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">pushButton&gt;setStyleSheet(&quot;QPushButton{background:transparent;}&quot;);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/menu (copy).png</normaloff>../../icons/menu (copy).png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="camera_BTN">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>105</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="whatsThis">
      <string>CAMERA</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchuser_b">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lastseen_l">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chilanka</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>last seen recently</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_b">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>500</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>sdfgsdfgfdsgdf</string>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="user_search_t">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>671</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>500</y>
       <width>671</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="usernamem_l">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chilanka</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mohammad Hossein Fadavi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="other_b">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>500</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>other</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_user_b">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="messegebox_t">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>510</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="cursorWidth">
      <number>2</number>
     </property>
     <property name="placeholderText">
      <string>   Type a massage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>510</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>581</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="side_bar_l">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>51</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">pushButton&gt;setStyleSheet(&quot;QPushButton{background:transparent;}&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../icons/menu (copy).png</normaloff>../../icons/menu (copy).png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="pv_LBL">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>5</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="doc_BTN">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>55</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>49</width>
       <height>49</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>record_b</zorder>
    <zorder>background_l</zorder>
    <zorder>attach_b_2</zorder>
    <zorder>call_b</zorder>
    <zorder>attach_b</zorder>
    <zorder>listWidget</zorder>
    <zorder>scrollArea</zorder>
    <zorder>lineEdit</zorder>
    <zorder>user_b</zorder>
    <zorder>emoji_BTN</zorder>
    <zorder>user_search_b</zorder>
    <zorder>camera_BTN</zorder>
    <zorder>searchuser_b</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>lastseen_l</zorder>
    <zorder>clear_b</zorder>
    <zorder>user_search_t</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
    <zorder>usernamem_l</zorder>
    <zorder>other_b</zorder>
    <zorder>menu_user_b</zorder>
    <zorder>messegebox_t</zorder>
    <zorder>send_b</zorder>
    <zorder>frame</zorder>
    <zorder>pv_LBL</zorder>
    <zorder>doc_BTN</zorder>
   </widget>
   <widget class="QFrame" name="emoji_FRM">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="emoji_bg_LBL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>671</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="send_b_12">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="emoji_BTN_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>513</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../icons/send2.png</normaloff>../../icons/send2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <zorder>mother_FRM</zorder>
   <zorder>setting_FRM</zorder>
   <zorder>emoji_FRM</zorder>
   <zorder>emoji_BTN_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
